src/core/dbus-execute.c

```diff
@@ -1152,6 +1152,7 @@ const sd_bus_vtable bus_exec_vtable[] = {
         SD_BUS_PROPERTY("PrivateUsers", "b", bus_property_get_bool, offsetof(ExecContext, private_users), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("PrivateMounts", "b", bus_property_get_tristate, offsetof(ExecContext, private_mounts), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("PrivateIPC", "b", bus_property_get_bool, offsetof(ExecContext, private_ipc), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("PrivatePIDs", "b", bus_property_get_bool, offsetof(ExecContext, private_pids), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
someone recently had to introduce PrivateUsersEx because bools don't scale in D-Bus ;-)

maybe out of paranoia start with a string on this one from the start? noone knows what it might be good for eventually.

Reply:
Touche, I'll make it a string

Reply:
not addressed yet.